examples/.experimental/next-app-dir/src/trpc/server-invoker.ts

```diff
@@ -13,9 +13,8 @@ export const api = experimental_createTRPCNextAppDirServer<typeof appRouter>({
     return {
       createContext: () => createContext('invoke'),
       contextSelector: (ctx, callOpts) => {
-        if (callOpts.path[0] === 'privateGreeting')
-          return [ctx.session?.user.id, ctx._userIdMock];
-        return [];
+        if (!['privateGreeting'].includes(callOpts.path[0])) return [];
```

Comment:
a comment would be nice - i don't know if even i would know what this does if i revisit this example in a month or 2 ;)